pkg/azuredisk/controllerserver.go

```diff
@@ -461,7 +461,11 @@ func (d *Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotRequ
 	klog.V(2).Infof("begin to create snapshot(%s) under rg(%s)", snapshotName, d.cloud.ResourceGroup)
 	future, err := d.cloud.SnapshotsClient.CreateOrUpdate(ctx, d.cloud.ResourceGroup, snapshotName, snapshot)
 	if err != nil {
-		return nil, status.Error(codes.Internal, fmt.Sprintf("create snapshot error: %v", err))
+		if strings.Contains(err.Error(), "existing") {
```

Comment:
what's the exact error? only matching `existing` word may be not enough

Reply:
[CreateSnapshot] error(compute.SnapshotsClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="BadRequest" Message="Changing property 'sourceResourceId' is not allowed for existing disk 'CreateSnapshot-snapshot-2'.")

Reply:
use `existing disk` word?